src/Language/PureScript/Interactive/Parser.hs

```diff
@@ -70,7 +70,7 @@ parseDirective cmd =
     Show     -> ShowInfo <$> parseReplQuery' (trim arg)
     Type     -> TypeOf <$> parseRest P.parseValue arg
     Kind     -> KindOf <$> parseRest P.parseType arg
-    Complete -> CompleteStr <$> Right arg
+    Complete -> CompleteStr <$> Right (trim arg)
```

Comment:
Rather than putting the `trim` here, I think it would be better to change the declaration of `arg` so that the space is already trimmed. How about `(dstr, arg) = second trim . break isSpace cmd`? The reason for this is that I don't think either extra spaces in between the directive and the argument or trailing spaces should be able to have any effect on the result, and that's for any directive, not just `:complete`.

Also, any expression of the form `f <$> Right x` is just `Right (f x)` so I think the latter is better style, as with the former you're kind of just making the reader jump through hoops. In this specific case I think `return (CompleteStr arg)` makes the most sense.